### PR TITLE
Fixed screensharing

### PR DESCRIPTION
**Pull request by Daboo1998**

### CODE REVIEW

File: src/scenes/collaboration/pages/ScreenSharePage/ScreenSharePage.tsx [hunk 1]
```diff
@@ -17,29 +17,33 @@ const ScreenSharePage: FC = () => {
...
+  const {videoTrack} = agoraScreenShareStore;
 
   const {t} = useTranslation();
   const history = useHistory();
 
   useEffect(() => {
     if (videoTrack) {
-      const agoraUserId = videoTrack?.getUserId() as string;
-      screenShareStore.setScreenOwner(agoraUserId);
+      if ('getUserId' in videoTrack) {
```
Comment (PAKovalev) [inline src/scenes/collaboration/pages/ScreenSharePage/ScreenSharePage.tsx:27]: `if (!!videoTrack?.getUserId) { ... }`

File: src/stores/MainStore/models/AgoraStore/AgoraScreenShareStore/AgoraScreenShareStore.ts [hunk 2]
```diff
@@ -131,9 +139,28 @@ const AgoraScreenShareStore = types
...
+      }
+    }),
+    stopScreenSharing() {
+      if (!self.videoTrack) {
+        return;
+      }
+
+      self.videoTrack.stop();
+
+      if ('close' in self.videoTrack) {
```
Comment (PAKovalev) [inline src/stores/MainStore/models/AgoraStore/AgoraScreenShareStore/AgoraScreenShareStore.ts:158]: `if (!!self.videoTrack?.close) {`

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: src/scenes/collaboration/pages/ScreenSharePage/ScreenSharePage.tsx (hunk 1 above)
Comment (Daboo1998) [reply src/scenes/collaboration/pages/ScreenSharePage/ScreenSharePage.tsx:27]: Won't work

File: src/stores/MainStore/models/AgoraStore/AgoraScreenShareStore/AgoraScreenShareStore.ts (hunk 2 above)
Comment (Daboo1998) [reply src/stores/MainStore/models/AgoraStore/AgoraScreenShareStore/AgoraScreenShareStore.ts:158]: Won't work

**Review by PAKovalev — APPROVED** [review]

**Pull request merged**